superset/security/manager.py

```diff
@@ -318,7 +320,7 @@ def get_datasource_access_error_object(
             },
         )
 
-    def get_table_access_error_msg(self, tables: Set["Table"]) -> str:
+    def get_table_access_error_msg(self, tables: Set[str]) -> str:
```

Comment:
Sadly this is somewhat of a half step back in order to move forward with standardizing the access patterns. Note long term I believe all these somewhat unpleasant msg/link methods will be deprecated in favor of the client leveraging the `SupersetError` logic which @etr2460 introduced.